Extension/package.nls.json

```diff
@@ -29,6 +29,13 @@
     "c_cpp.command.BuildAndRunFile.title": "Run C/C++ File",
     "c_cpp.command.AddDebugConfiguration.title": "Add Debug Configuration",
     "c_cpp.command.GenerateDoxygenComment.title": "Generate Doxygen Comment",
+    "c_cpp.command.addSshTarget.title": "Add SSH target",
+    "c_cpp.command.removeSshTarget.title": "Remove SSH target",
+    "c_cpp.command.setActiveSshTarget.title": "Set this SSH target as the active target",
+    "c_cpp.command.selectActiveSshTarget.title": "Select an active SSH target",
+    "c_cpp.command.selectSshTarget.title": "Select SSH target",
+    "c_cpp.command.activeSshTarget.title": "Get the currently active SSH target",
```

Comment:
Should "currently" be changed to "current"? Or maybe remove it if there is always only one active SSH target? If so, "Get the active SSH target" would be sufficient?

Reply:
Should be "currently", it's an adv. for "active", an adj. But I agreee that "Get the active SSH target" is sufficient.